dsc/tests/dsc_export.tests.ps1

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'resource export tests' {
+    
+    It 'Export can be called on individual resource' {
+
+        $processes = dsc resource export -r Microsoft/Process
+        $LASTEXITCODE | Should -Be 0
+        $processes.count | Should -BeGreaterThan 1
+    }
+
+    It 'Export can be called on a configuration' {
+
+        $json = @'
```

Comment:
To avoid confusion, this should be `$yaml`

Reply:
Updated.